src/routes/Jarjestajat/Jarjestaja/Hakemukset/Muutospyynto/components/MuutospyyntoWizardMuut.js

```diff
@@ -259,26 +262,43 @@ class MuutospyyntoWizardMuut extends Component {
             if (kuvaus && kuvaus !== '') {
 
               return (
-                <div>
+                <div key={identifier}>
                   { koodiArvo === "2" && 
                     <p>{MUUTOS_WIZARD_TEKSTIT.MUUTOS_MUUT.YKSIVALINTA.FI}:</p> 
                   }
                   { koodiArvo === "3" && 
                     <p>{MUUTOS_WIZARD_TEKSTIT.MUUTOS_MUUT.LISAALINTA.FI}:</p> 
                   }
                   <CheckboxRowContainer key={identifier} className={customClassName}>
-                    <Checkbox>
-                      <input
-                        type="checkbox"
-                        id={identifier}
-                        checked={isChecked}
-                        disabled={voimassaLoppuPvm !== undefined}
-                        onChange={(e) => {
-                          handleCheckboxChange(e, editValues, fields, isInLupa, muu)
-                        }}
-                      />
+                    
+                  { koodiArvo === "2" || koodiArvo === "16" || koodiArvo === "17" || koodiArvo === "18"
```

Comment:
Koodiarvoja ei oikein voida "kovakoodata" tännekään, koska osa voi poistua käytöstä tai niitä voi tulla lisää. Tässä tilanteessa nämä koodiarvot 2, 16-21 liittyvät kaikki vaativaan erityiseen tukeen. Voidaanko frontissa ajaa tarkistus nyt sitä vasten? Eli sieltä koodistosta palautuu näille käsite-arvo "vaativa". Jos tehtäisiin reactin sisäinen taulukko, jossa nämä koodiarvot säilytettäisiin ja verrattaisiin näissä ehdoissa siihen? Tällöin tarkistuksesta saataisiin dynaaminen? Ehdotuksia? Ideoita?

Reply:
Koska nämä kaikki ovat vaativia, en keksinyt muuta eroa erotella 3 ja 12. Mutta toki taulukkoa voisi käyttää, jos se on yhtään parempi tapa.